### PR TITLE
Replace AY SuSEFirewall2 config by Firewalld

### PR DESCRIPTION
**Pull request by teclator**

### CODE REVIEW

File: xml/ay_bigfile.xml [hunk 1]
```diff
@@ -12993,32 +12993,23 @@ exit 0
...
-    SUSE Firewall can be configured using the <literal>firewall</literal>
-    resource. All the properties in this resource are optional and all of them
-    are of type text (except the boolean
-    <literal>start_firewall</literal> and <literal>enable_firewall</literal> properties).
-   </para>
+     SuSEfirewall2 has been replaced by <literal>firewalld</literal> since
+     &slsa; 15. Profiles using SuSEfirewall2 properties will be translated
+     to <literal>firewalld</literal> but only some of them will be really
+     supported. Refer to <xref linkend="sec.security.firewall.firewalld"/>
+     if you want to know details about <literal>firewalld</literal>.
```
Comment (imobachgs) [inline xml/ay_bigfile.xml]: As it is not `firewalld` documentation, I would add *about firewalld support*. Or just "refer to ... for further details.".

File: xml/ay_bigfile.xml [hunk 2]
```diff
@@ -12993,32 +12993,23 @@ exit 0
...
 
    <sect2 xml:id="CreateProfile.firewall.general">
     <title>General Firewall Configuration</title>
 
     <para>
-     The following properties are intended to configure the general settings of
-     SUSE Firewall. Most of them are a direct representation of the corresponding
-     setting at <filename>/etc/sysconfig/SuSEfirewall2</filename>. Check the
-     comments in that file for further information.
+      In <literal>firewalld</literal> the general configuration only expose few
```
Comment (imobachgs) [inline xml/ay_bigfile.xml]: *exposes*

File: xml/ay_bigfile.xml [hunk 3]
```diff
@@ -13072,235 +13063,58 @@ exit 0
...
          <para>
-          yes / no / notrack
+           Type of dropped packages to be logged
          </para>
         </entry>
         <entry>
-         <para>
-          Whether to protect the firewall from the internal network.
+          <para>
+            Enable logging dropped packages for the type selected.
```
Comment (imobachgs) [inline xml/ay_bigfile.xml]: logging *of*... (not sure at all).

**Comment by svenseeberg** [comment]
Continued in #239

**Pull request closed without merging**